lib/active_fedora/file.rb

```diff
@@ -139,6 +140,19 @@ def metadata?
       false
     end
 
+    def external_content?
+      external_url.present?
+    end
+
+    def external_url
+      return nil unless mime_type.start_with? "message/external-body"
+      mime_type[/url=['"](.+?)['"]/i, 1]
+    end
+
+    def external_url=(external_file_url)
+      self.mime_type = "message/external-body; access-type=URL; URL=\"#{external_file_url}\""
```

Comment:
I'd have to check to make sure, but I think you could store content but you wouldn't be able to retrieve it.  But once you changed the mime type to something else, then you could probably retrieve the content again.